src/main/java/Deadline.java

```diff
@@ -0,0 +1,18 @@
+public class Deadline extends Task {
```

Comment:
the websites says: "Class variables should never be declared public"
So, perhaps it should not public.